secvarctl.c

```diff
@@ -123,7 +123,7 @@ static struct backend *getBackend()
 		prlog(PR_WARNING, "WARNING!! Could not extract data from %s , assuming platform does not support secure variables\n", secVarFormatLocation);
 		goto out;
 	}
-	//loop through all known backends
+	// loop through all known backends
```

Comment:
Don't include Jenkins-related commits in the history here. If you want to keep this change, have the commit message reflect what is actually changing in the commit.